assets/monitoring/prometheus/v1/scylladbmanager.servicemonitor.yaml

```diff
@@ -4,14 +4,14 @@ metadata:
   name: scylla-manager-service-monitor
   namespace: scylla-manager
 spec:
-  jobLabel: "app.kubernetes.io/name"
+  jobLabel: app.kubernetes.io/name
   selector:
     matchLabels:
       app.kubernetes.io/name: scylla-manager
```

Comment:
i think this would need to be configurable

Reply:
Sure, that was only a WIP. Dropping the manager support until we have an automated manager setup as agreed.